users.js

```diff
@@ -95,6 +96,86 @@ var lockedUsers = Object.create(null);
 var lockedRanges = Users.lockedRanges = Object.create(null);
 var rangelockedUsers = Object.create(null);
 
+function importLocksBans() {
+	fs.readFile('config/locksbans.csv', function (err, data) {
+		if (err) return;
+		var now = Date.now();
+		data = ('' + data).split("\n");
+		for (var i = 0; i < data.length; i++) {
+			if (!data[i]) continue;
+			var row = data[i].split(",");
+			var expiration = parseInt(row[2]);
+			if (now > expiration) continue;
+			var entry = [row[1], expiration];
+			if (row[0] === 'banip') {
+				bannedIps[row[3]] = entry;
+			} else if (row[0] === 'ban') {
+				bannedUsers[row[3]] = entry;
```

Comment:
I'm not sure what you mean. When the file is exported, the first values on each line (<code>row[0]</code> here) differentiate locked/banned IPs and usernames because they are stored in different objects.